cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,71 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can use `roxctl` CLI, Jenkins, and API to scan images stored in a cluster specific {ocp} integrated image registry. To do so, specify the appropriate cluster using the delegated scanning configuration or the cluster parameter in `roxctl` CLI, Jenkins, and API.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images in a remote cluster
+
+To scan an image in a remote cluster, use the `roxctl image scan` command. Specify the location and name of the image, and specify the remote cluster. You can also specify optional parameters to customize the behavior of the command.
```

Comment:
```suggestion
== Scanning images via a remote cluster

To scan images from cluster local registries via a remote cluster, specify the appropriate cluster using the delegated scanning configuration or use the cluster parameter described next.
```

Reply:
Since this section is specific to the `CLI`, may be best to remove references to the API / Jenkins.

Also, may be good to link to the delegated scanning section of the docs.